crud/common/utils.lua

```diff
@@ -97,13 +98,23 @@ end
 
 function utils.get_space(space_name, replicasets)
     local replicaset = select(2, next(replicasets))
+
+    if replicaset.master.conn.error ~= nil then
```

Comment:
Can you be sure that replicaset always has a master and master always has a non-nil connection?

Reply:
At the same time, it will not be difficult to add a check `replicaset.master.con ~= nil` and `replicaset ~= nil`, just in case. ( `<attempt to index local 'replicaset' (a nil value)>` error occurs, for example, when we only have a router)

Reply:
> At the same time, the need of `replicaset.master.con ~= nil`checking is not entirely obvious to me, vshard implementation use net_box.connect that does not simply return `nil` in case of an unsuccessful connection

Ok, I think we may rely on this.

> AFAIU, checking for replicaset.master ~= nil will be implemented in the https://github.com/tarantool/crud/issues/95.

I don't think it should. Either there is something similar to the above one ("vshard guarantees that there is always a `master`") or we should check it explicitly. Otherwise, it would be still possible to get an unclear error similar to #331 one.

Reply:
> `replicaset ~= nil`

Let's add this one.

Reply:
Done, added check `replicaset ~= nil`

Reply:
I'm not an expert in vshard pipeline, but I think it's theretically possible to get a `nil` master. Let's checks this one too.

https://github.com/tarantool/vshard/blob/f5f386a5a35e6e5efd8f4f2ed1b3d208fdae9095/vshard/replicaset.lua#L223-L241

Reply:
Done, added check `replicaset.master ~= nil`
Also updated `CHANGELOG.md` and commit name/description